accessibility/accessibility-in-pdfs.md

```diff
@@ -0,0 +1,92 @@
+
+The [Web Accessibility Initiative](https://www.w3.org/WAI/) has produced an alternative text decision tree to help you decide whether an image is decorative or meaningful: [An alt decision tree](https://www.w3.org/WAI/tutorials/images/decision-tree/)
+
+#### Links
+Links must have concise, meaningful, descriptive text so that screen reader users can understand their purpose. Screen reader users don’t always read the whole page in context - they may jump from link to link without reading the surrounding text. 
+
+Avoid link text that says “click here” or “more” - be more specific. Focus on where the link is going, and try to describe it as concisely as you usefully can. 
+
+WebAIM has published guidance on [what good link text looks like](https://webaim.org/techniques/hypertext/link_text) (the final section on “hover and focus effects” is for websites only, and isn’t relevant to PDF documents).
+
+#### Contrast
+Text must have enough contrast with the background colour in order to be visible and readable by as many users as possible. Text contrast is measured in a ratio and at least a minimum 4.5:1 ratio must be achieved. Use a tool to check your contrast ratio, like the [WebAIM Contrast Checker](https://webaim.org/resources/contrastchecker/).
+
+#### Colour
+Don’t use colour as the only way of conveying information (for example colouring some text in red to indicate a problem, but having no other way of indicating that the problem exists). Blind or low vision user may not be able to see your colours well or at all, and some colourblind users may be unable to distinguish between certain colours. 
+
+#### Built-in Structure and Navigation 
+Order lists with bullet points or numbers using the built-in list creation tools in your word processor. 
+
+Include a table of contents, and add page numbers.
+
+#### Tables
+Microsoft Word can only help to make basic tables accessible. You must manually tag these after the document has been created (see [Fixing non-accessible PDFs below](#how-to-fix-non-accessible-PDFS?)). 
+
+Google documents do not have the ability to mark up tables correctly, but the Grackle Docs addon allows you to remediate tables once you’ve built them. 
+
+#### Rich media 
+If your PDF contains rich media files like audio or video, it must also include alternatives so that users who can’t see or hear the media can still understand what it conveys. 
+
+Depending on the rich media you use, this may involve [transcripts, closed captions, and/or audio description](https://www.w3.org/WAI/media/av/). 
+
+#### Forms
+Consider whether you really need to use a PDF file for your purposes - can you achieve the same results with a webpage, or with a survey tool like Google Forms or SurveyMonkey? 
+
+If you have no other choice, you’ll need to add your form fields to your PDF manually with Adobe Acrobat Pro DC. See WebAIM’s guide to [Accessible Forms in Acrobat](https://webaim.org/techniques/acrobat/forms) for more details. 
```

Comment:
> If you have no other choice, you’ll need to add your form fields to your PDF manually with Adobe Acrobat Pro DC.

It's not clear to me from this sentence if Acrobat is required to make the forms accessible, or if it's required to create any forms at all.